DQM/SiTrackerPhase2/python/Phase2ITMonitorRecHit_cff.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
+from DQM.SiTrackerPhase2.Phase2ITMonitorRecHit_cfi import Phase2ITMonitorRecHit 
```

Comment:
I prefer to keep this file since we might need to update binning, ranges etc. Moreover, there is an option of switching a histogram on and off which will be needed.